### PR TITLE
feat: :sparkles: add third file

### PR DESCRIPTION
**Pull request by jonp-h**
add third text file with petname

refs: #2

### CODE REVIEW

File: thirdfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Rex
```
Comment (knuteivi) [inline thirdfile.txt]: Rex was a bad name! FIX!

**Review by knuteivi — CHANGES_REQUESTED** [review]
Look at comments! FIXXXXX!!!!!!

File: thirdfile.txt (hunk 1 above)
Comment (jonp-h) [reply thirdfile.txt]: fixed in last commit!

**Review by knuteivi — APPROVED** [review]
GTM (good to merge)

**Pull request merged**